cmake/Util.cmake

```diff
@@ -193,28 +192,26 @@ endfunction()
 # Passing CMAKE_BUILD_TYPE for multi-config generators will not only not be
 # used, but also generate a warning for the user.
 function(set_default_buildtype BUILD_TYPE)
-  set(allowableBuildTypes Debug Release MinSizeRel RelWithDebInfo)
-  if(NOT BUILD_TYPE IN_LIST allowableBuildTypes)
+  set(defaultBuildTypes Debug Release MinSizeRel RelWithDebInfo)
+  if(NOT BUILD_TYPE IN_LIST defaultBuildTypes)
     message(FATAL_ERROR "Invalid build type: ${BUILD_TYPE}")
```

Comment:
Won't this still fail?

Reply:
Nah, this is only when calling the function eith a non-default build type. Tbh unsure if this is still needed though. We can probably remkve this

Reply:
Isn't `None` a non-default build type? :) Or am I missing something?

Reply:
Ah, I was. This is about the default build type (that the project opts in to) not the specific build type the user is selecting.

Reply:
Yeah, that's what I tried to say lol. I don't think this is needed, so will remove this as well.